### DISCUSSION TITLE
How to merge from main branch to my local branch?

### DISCUSSION THREAD

**Discussion by qincheng0408** [fullstackdev48/shopify-template-ruby#7] [Q&A] [answered]
I have 2 branches in my local: main and development. and I am going to merge the codes from origin main to my local development branch. How can I do this using Git Command?

**Comment by fullstackdev48** [comment, marked as answer]
Good question. You can do this following steps.
1. $ git checkout development
2. $ git pull origin main
That's all. I hope this will solve your issue!

**Reply by qincheng0408** [reply]
It works for me, Thanks!

**Comment by baby1194** [comment]
I think it is okay
